### PR TITLE
[release-2.4] Organization rename to `stolostron`

### PR DESCRIPTION
**Pull request by dhaiducek**
- stolostron/backlog#18776

I was having issues changing the org name because it appears the propagator package hadn't been updated in a while, so updating it caused dependency issues. ~Cherry-picking the operator-sdk changes seems to have fixed it.~

### CODE REVIEW

File: pkg/controller/sync/policy_spec_sync.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ type ReconcilePolicy struct {
 // Note:
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcilePolicy) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
```
[the remark this answers is not in the document]
Comment (ycao56) [reply pkg/controller/sync/policy_spec_sync.go:102]: `ctx context.Context` param is not used below. I am not sure if it matters or not. @mprahl do you know if `ctx.TODO()` and `context.TODO()` are same?

[the remark this answers is not in the document]
Comment (mprahl) [reply pkg/controller/sync/policy_spec_sync.go:102]: The solution would be to use the `ctx` variable everywhere there is `context.TODO()` specified below. That way if the input context is canceled by the manager, the API requests would presumably also get canceled.

That being said, why does the controller-runtime and related packages need to be updated in 2.4? Unless there is a security or bug fix this addresses, I'd rather be conservative and not upgrade any of the packages.

[the remark this answers is not in the document]
Comment (dhaiducek) [reply pkg/controller/sync/policy_spec_sync.go]: This was originally pointing to a propagator version from April, so pointing to `release-2.3` seemed reasonable and did the trick. Thanks for the tips!

**Comment by ycao56** [comment]
We can go ahead and merge this one. Once policy-propagator `release-2.3` branch removes its dependency on [github.com/open-cluster-management/api v0.0.0-20200610161514-939cead3902c](https://github.com/stolostron/governance-policy-propagator/blob/release-2.3/go.mod#L9), another update is needed.

**Review by ycao56 — APPROVED** [review]

**Pull request merged**